src/pages/projects.js

```diff
@@ -544,6 +544,37 @@ const ProjectDescriptions = () => (
       </div>
     </div>
 
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">KubeArmor</TitleWithAnchor>
+      <div className="project-minor-title">Container-aware Runtime Security Enforcement System</div>
+      <div className="project-body">
+        <a
+          className="project-logo"
+          target="_blank"
+          href="https://github.com/accuknox/KubeArmor"
+        >
+          <img src={require("../assets/projects-logos/kubearmor.png")} />
+        </a>
+        <div className="project-description">
+          <p>
+	    <a href="https://kubearmor.com">
+	      <b>Website</b>
+	    </a>{" "}
+	    |{" "}
+            <a href="https://github.com/accuknox/KubeArmor">
+              <b>GitHub</b>
+            </a>{" "}
+          </p>
+          <p>
+	  KubeArmor a container-aware runtime security enforcement system that 
```

Comment:
The verb is missing in clause.

Reply:
(Note also that the projects were sorted in alphabetical order so far, this is not the case for this addition.)

Reply:
Thanks for the comments.
I added the missing verb ("is") and sorted the projects in alphabetical order.

Reply:
Is there anything that I need to do for the merging process?